torch/_prims/__init__.py

```diff
@@ -773,6 +775,44 @@ def _mul_nvfuser(fd: Any, a: TensorLikeType, b: TensorLikeType):
 #
 # View operations
 #
+# TODO: model view relationships
+# TODO: model storage
+def _as_strided_meta(
+    a: TensorLikeType, size: ShapeType, stride: StrideType, storage_offset: int
+) -> TensorLikeType:
+    assert len(size) == len(stride)
```

Comment:
should these be RuntimeErrors?

Reply:
Yep -- I'll fix them up along with other lingering asserts in a quality-focused follow-up